docs/jobs_reference.rst

```diff
@@ -134,6 +135,9 @@ Jobs
                                      cannot be scheduled because the lack of computation
                                      cluster resources (memory, CPU/GPU etc).
 
+      :param float life_span: job run-time limit in the format '1d2h3m4s'
```

Comment:
The `'1d2h3m4s'` is invalid float.
What is `None` for? I assume it is for server default, please correct me if I'm wrong.

Reply:
oops

Reply:
`None` to disable. I'd actually leave the default API value for `1d`, not for `None` as it is now.

Reply:
Why not enforce the default on server side?
It is more reliable.
I feel potential problems if we want to change 1day to another default eventually.

Reply:
Long time ago, when I was implementing this feature [on the backend](https://github.com/neuromation/platform-api/pull/1044), my very first version enforced default server-side as you suggest. However, after several review we agreed to pass default value by the client so that:
1. no problems with DB migration, or if we decide not to write the default value to the database, then no implicit logic;
2. much less code
3. not over-complicated protocol (`0` means `limit=0`)
4. easier to change the default value per-user: no need to put it to the database.